kolibri/plugins/learn/assets/src/views/LibraryPage/OtherLibraries.vue

```diff
@@ -257,4 +260,11 @@
     margin-left: 0.75em;
   }
 
-</style>
+  .connection-status{
+    margin-bottom: 30px;
+    margin-top: -15px;
```

Comment:
The negative `margin-top` here combined with the negative `margin-bottom` on line 7 makes the connection status text too close to the text above it and is inconsistent with the spacing of the text below it, removing the negative margins and perhaps making the `margin-bottom` in this class smaller should even it out.

![ConnectionStatusSpacing](https://github.com/user-attachments/assets/637e5a74-9981-4341-9cd8-9a20de8481c3)